pontoon/base/static/js/main.js

```diff
@@ -485,28 +481,41 @@ $(function() {
       };
     }
 
-    var index = $(this).index(),
-        current = $(this).attr('class'),
+    function getDate(el) {
+      var date = $(el).find('time').attr('datetime') || 0;
+      return new Date(date).getTime().toString();
+    }
+
+    function getString(el) {
+      return $(el).find('span:eq(' + index + ')').html();
+    }
+
+    var node = $(this),
+        index = $(this).index(),
```

Comment:
You can replace `$(this)` with `node` and save yourself a ton of re-jQuerying things throughout this function.